src/BenchmarkDotNet/Environments/ProcessorBrandStringHelper.cs

```diff
@@ -10,13 +10,18 @@ public static class ProcessorBrandStringHelper
         /// Transform a processor brand string to a nice form for summary.
         /// </summary>
         /// <param name="processorName">Original processor brand string</param>
+        /// <param name="frequency">processor actual frequency</param>
         /// <returns>Prettified version</returns>
         [NotNull]
-        public static string Prettify([NotNull] string processorName)
+        public static string Prettify([NotNull] string processorName, double? frequency = null)
```

Comment:
It should be renamed to maxFrequency; `Frequency?` should be used instead of `double?`